metrics/network/lib/network-test-common.bash

```diff
@@ -36,7 +36,7 @@ function start_server {
 	local server_name="$1"
 	local image="$2"
 	local server_command="$3"
-	$DOCKER_EXE run -d --runtime="${RUNTIME}" --name=${server_name} ${image} \
+	$DOCKER_EXE run -d --runtime="${RUNTIME}" --privileged --name=${server_name} ${image} \
```

Comment:
As I believe we only need this for the one test, and you fix the 'client' call by adding the priv to the `extra_args` - maybe it would be better if we add an `extra_args` argument to the start_server func like we have in the client, and then add the priv to the iperf3 test code only? I think that make the code more uniform, and keeps the 'special case' code in one place (in the relevant test).

Reply:
make sense, thanks